This huge PR adds proper SEO to all articles in the docs for better search engine indexing (#162)
golemfactory/golem-docs: src/pages/docs/creators/javascript/examples/working-with-images.md

---

> title: Guide to Utilizing Custom Golem Images in Requestor Scripts

Reviewer: ```suggestion
title: Examples of utilizing Custom Golem Images in Requestor Scripts
```
